formats/csv.js

```diff
@@ -1,16 +1,20 @@
 module.exports.type = "csv";
+var DEFAULT_DELIMITER = ','
 
-module.exports.is = function isCSV( sample ) {
-    return sample.toString().split( "\n" )[ 0 ].indexOf( "," ) != -1;
+module.exports.is = function isCSV( sample, delimiter ) {
```

Comment:
Agree, will change the whole thing to accept options instead of specific parameter